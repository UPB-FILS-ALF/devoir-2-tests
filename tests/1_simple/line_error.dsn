& LINE error

LINE:  120 120 line
POSITION: 100 100
LINE:  100 100 location