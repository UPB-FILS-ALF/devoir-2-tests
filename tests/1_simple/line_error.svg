<?xml version="1.0" encoding="UTF-8" ?>
<svg xmlns="http://www.w3.org/2000/svg" version="1.1" width="1024" height="768">
	<!-- position 100, 100 -->
	<line x1="100" y1="100" x2="100" y2="100" stroke="rgb(0,0,0)"  stroke-width="1" />
</svg>
