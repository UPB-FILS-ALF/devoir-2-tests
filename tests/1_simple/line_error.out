ERROR LINE (3): LINE parameter 3 needs to be one of (location, polar), you wrote line
